sdk/tables/Azure.Data.Tables/tests/TableEntityTests.cs

```diff
@@ -60,26 +59,66 @@ public void DynamicTableEntityGetWrongTypesThrows()
             Assert.That(() => entityWithAllTypes.GetInt64("binary"), Throws.InstanceOf<InvalidOperationException>(), "GetInt64 should validate that the value for the inputted key is a Int64.");
         }
 
+        /// <summary>
+        /// Validates getting properties involving null.
+        /// </summary>
+        [Test]
+        public void DynamicEntityGetNullOrNonexistentProperties()
+        {
+            // Test getting new property works.
+            Assert.IsFalse(entityWithoutPK.TryGetValue(TableConstants.PropertyNames.PartitionKey, out _));
```

Comment:
nit: Would you be so kind as to use the below style of asserts throughout these tests? They are much more descriptive in their output.

```c#
Assert.That(entityWithoutPK.TryGetValue(TableConstants.PropertyNames.PartitionKey, out _), Is.False())
```